bindings/core/src/response.rs

```diff
@@ -85,9 +85,6 @@ pub enum Response {
     /// - [`GetProtocolParameters`](crate::method::ClientMethod::GetProtocolParameters)
     ProtocolParameters(ProtocolParametersDto),
     /// Response for:
-    /// - [`PrepareTransaction`](crate::method::ClientMethod::PrepareTransaction)
-    PreparedTransactionData(PreparedTransactionDataDto),
-    /// Response for:
     /// - [`SignTransaction`](crate::method::ClientMethod::SignTransaction)
     SignedTransaction(PayloadDto),
```

Comment:
SignedTransaction from clinet is gone, should be the one in  SecretManager

Reply:
Updated the comment